code/modules/mob/living/carbon/human/species_types/dullahan.dm

```diff
@@ -0,0 +1,116 @@
+/datum/species/dullahan
+	name = "dullahan"
+	id = "dullahan"
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,NOBREATH,NOHUNGER)
+	mutant_bodyparts = list("tail_human", "ears", "wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	use_skintones = TRUE
+	mutant_brain = /obj/item/organ/brain/dullahan
+	mutanteyes = /obj/item/organ/eyes/dullahan
+	mutanttongue = /obj/item/organ/tongue/dullahan
+	mutantears = /obj/item/organ/ears/dullahan
+	blacklisted = TRUE
+	limbs_id = "human"
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+
+	var/obj/item/bodypart/head/myhead
+
+/datum/species/dullahan/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
+	. = ..()
+	var/obj/item/bodypart/head/head = H.get_bodypart("head")
+	if(head)
+		myhead = head
+		head.drop_limb()
+		myhead.flags_1 = HEAR_1
+		myhead.throwforce = 25
+		new /obj/item/dullahan_relay (myhead, H)
+
+/datum/species/dullahan/spec_life(mob/living/carbon/human/H)
+	if(myhead)
+		update_vision_perspective(H)
+
+		var/turf/Me = get_turf(H)
+		var/turf/Head = get_turf(myhead)
+		if(Me == Head || (Head in oview(7, Me)))
+			H.disabilities &= ~DEAF
+		else
+			H.disabilities |= DEAF
+	else
+		H.gib()
+	var/obj/item/bodypart/head/head2 = H.get_bodypart("head")
+	if(head2)
+		H.gib()
+
+/datum/species/dullahan/proc/update_vision_perspective(mob/living/carbon/human/H)
+	var/obj/item/organ/eyes/eyes = H.getorganslot(ORGAN_SLOT_EYES)
+	if(eyes)
+		H.update_tint()
+		if(eyes.tint)
+			H.reset_perspective(H)
+		else
+			H.reset_perspective(myhead)
+
+/obj/item/organ/brain/dullahan
+	decoy_override = TRUE
+	vital = FALSE
+
+/obj/item/organ/tongue/dullahan
+	zone = "abstract"
+
+/obj/item/organ/tongue/dullahan/TongueSpeech(var/message)
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		if(H.dna.species.id == "dullahan")
+			var/datum/species/dullahan/D = H.dna.species
+			D.myhead.say(message)
+	message = ""
+	return message
+
+/obj/item/organ/ears/dullahan
+	zone = "abstract"
+
+/obj/item/organ/eyes/dullahan
+	name = "head vision"
+	desc = "An abstraction."
+	actions_types = list(/datum/action/item_action/organ_action/dullahan)
```

Comment:
no ui_action_click!!!! REEEEE

Reply:
I did it how the other organs did it how am I supposed to be doing it

Reply:
@Cyberboss halloween event is soon!

Reply:
There's no ui_click in here ? What's the problem.

Reply:
This is not an issue.
defined her is ```/datum/action/item_action/organ_action/dullahan```, which is a custom action type, with its own ```Trigger()``` proc.
It is NOT the ```ui_action_click``` I put a bounty on the head of.

That horrific mess looks like a ```Topic()``` proc, but grosser.